sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/Statsbeat/Statsbeat.cs

```diff
@@ -119,90 +146,54 @@ private static string GetOS()
             return "unknown";
         }
 
-        internal static void InitializeAttachStatsbeat(string connectionString)
+        internal static string? GetStatsbeatConnectionString(string ingestionEndpoint)
         {
-            // check whether it is disabled or already initialized.
-            if (s_isEnabled && s_attachStatsBeatMeterProvider == null)
+            var patternMatch = s_endpoint_pattern.Match(ingestionEndpoint);
+            string? statsbeatConnectionString = null;
+            if (patternMatch.Success)
             {
-                if (s_statsBeat_ConnectionString == null)
+                var endpoint = patternMatch.Groups[1].Value;
+                if (s_eU_Endpoints.Contains(endpoint))
```

Comment:
nit: same here, try using EU or change the variable name in line 160 to  match this format